### PR TITLE
[StyleCop] Fix warnings in Japanese DatePeriodParser

### PR DESCRIPTION
**Pull request by Bill7zz**
- Add trailing coma in multi-line initializers
- Use String.Empty instead empty strings
- Add whitespace before * operator
- Remove trailing whitespace

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/DatePeriodParser.cs [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 {
     public class DatePeriodParser : IDateTimeParser
     {
-        private const int LastMonthOfYear = 12;
+        public static readonly string ParserName = Constants.SYS_DATETIME_DATEPERIOD; // "DatePeriod";
 
-        public static readonly string ParserName = Constants.SYS_DATETIME_DATEPERIOD; //"DatePeriod";
+        private const int LastMonthOfYear = 12;
 
         private static readonly IDateTimeExtractor SingleDateExtractor = new DateExtractor();
```
Comment (Aliandi) [inline .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/DatePeriodParser.cs:16]: All static properties should be togther, shouldn't them?

**Review by Aliandi — COMMENTED** [review]
Reorder maybe?

**Review by Aliandi — CHANGES_REQUESTED** [review]
Check logic changes

EDIT: my bad!

File: .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/DatePeriodParser.cs (hunk 1 above)
Comment (Bill7zz) [reply .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/DatePeriodParser.cs:16]: The order according to StyleCop rules is First all public variables static or not static then protected finally private in the same way.

Comment (Aliandi) [reply .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/DatePeriodParser.cs:16]: ACK

**Review by Aliandi — APPROVED** [review]

**Review by VictorGrycuk — APPROVED** [review]
LGTM

**Review by JuanAr — APPROVED** [review]
👍

**Pull request closed without merging**